tests/Feature/Checkout/CheckoutTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
```

Comment:
@drewroberts @sl0wik @joshtorres  - this test file, along with the supporting `TestSellable` implementation in this package provide some code examples of how to interact with checkout using the CartInterface / CartItemInterface.    This first set covers getting things into the cart, including an example where a single UX action results in multiple cart items (ie, the "Fee" scenario). 

Coming next are examples of completing checkout with and without payment.  These will get into the types of events raised during the conversion and what types of things you'd typically in response to each event.

After that, examples of discount & voucher handling.

Reply:
Can you add a little more details as to why there will ever be a checkout without a payment? IS that just to make the package more versatile?

Reply:
Can you add a little more details as to why there will ever be a checkout without a payment? Is that just to make the package more versatile?

Reply:
Many businesses allow purchase w/o pre-payment - balance due is invoiced in some manner.  I also thought you wanted to use same checkout flow in Nova and I thought I heard you say those are invoiced (or maybe you want those to be invoiced).  But, the package separation kind of leads to this approach as well -- its only the App builder that knows how payment will be handled, not checkout.

Reply:
Perfect. Thanks for reminding me and I agree.